prow/plugins/approve/approve.go

```diff
@@ -47,9 +47,9 @@ const (
 )
 
 var (
-	associatedIssueRegex = regexp.MustCompile(`(?:kubernetes/[^/]+/issues/|#)(\d+)`)
-	commandRegex         = regexp.MustCompile(`(?m)^/([^\s]+)[\t ]*([^\n\r]*)`)
-	notificationRegex    = regexp.MustCompile(`(?is)^\[` + approvers.ApprovalNotificationName + `\] *?([^\n]*)(?:\n\n(.*))?`)
+	associatedIssueRegexFormat = `(?:%s/[^/]+/issues/|#)(\d+)`
```

Comment:
Adding a link to the PR itself (by saying #XXXX, where #XXXX is the PR number) will bypass the restriction....

Reply:
Or I guess it's not a bug unless it's documented? /s